src/components/atoms/videoInfo/VideoInfo.tsx

```diff
@@ -1,16 +1,20 @@
-import { Typography } from "@mui/material";
-import { ReactNode } from "react";
+import { Typography, Link } from '@mui/material'
+import { ReactNode } from 'react'
 
 type VideoInfoProps = {
-  label?: string;
+  label?: string
   infoValue: ReactNode
-};
+}
 
 export function VideoInfo({ label, infoValue }: VideoInfoProps) {
   return (
     <Typography>
       {!!label && <b>{`${label}: `}</b>}
-      {infoValue}
+      {label === 'Client' ? (
+        <span>{infoValue}</span>
+      ) : (
+        <Link href={`/${infoValue}`}>{infoValue}</Link>
```

Comment:
Are we passing in a `ReactNode` here? Could that end up being the link? I might be confused here about what we're passing in as this prop.